lib/console_web/controllers/router/device_controller.ex

```diff
@@ -443,5 +443,11 @@ defmodule ConsoleWeb.Router.DeviceController do
     else
       conn |> send_resp(200, "")
     end
+
+    removed_devices = Devices.get_devices_in_list(removed_device_ids)
```

Comment:
would we want to do this after the condition so we don't do a db get if unnecessary?

Reply:
we have to do this to check if the removed_device_ids router sends over exist, the check in the next line is to make sure that this query returns 0 items